primitives/src/lib.rs

```diff
@@ -529,10 +529,15 @@ impl Asset {
 #[allow(clippy::unnecessary_cast)]
 pub enum CurrencyId {
 	#[default]
+	#[serde(rename = "Native")]
```

Comment:
This changes may be significant. The particular reason I added these (and removed the `camelCase` renaming above, is now due to the following panic:
```
Client(Storage("Invalid JSON blob: unknown variant `xCM`, expected one of `Native`, `XCM`, `Stellar`, `ZenlinkLPToken`, `Token` at line 1 column 2105"))
```

This comes from the changes in how the chain spec is defined now (first serialized in a json format).
I believe this error was also an issue for some extrinsics.